src/main/resources/application.yaml

```diff
@@ -10,7 +10,7 @@ kship:
 
 logging:
   level:
-    root: INFO
```

Comment:
@Buwujiu  is that intended to go on a release ?

Reply:
Added Alexis to reviewers too.